### ISSUE TITLE
Nightly builds on download page

### ISSUE THREAD

**Issue by silverwind** [nodejs/nodejs.org#510]
[Nightly builds](https://nodejs.org/download/nightly/) are very hard to find right now and not even reachable through directory navigation from [nodejs.org/dist](https://nodejs.org/dist). Can we agree to feature them more prominently on the [downloads](https://nodejs.org/en/download/) page, maybe in a small paragraph below the table?

**Comment by fhemberger** [comment]
Good catch, let's do this. :+1:

**Comment by rvagg** [comment]
We'll have a new line that can go with this too soon, name uncertain but it'll be `master` on perhaps a weekly schedule with slightly less scary version strings than nightlies https://github.com/nodejs/node/issues/4856

**Comment by silverwind** [comment]
Oh, and we probably should mention that semver issue with npm warning because of the version string not matching.

@rvagg would you recommend linking to the current nightlies, or should we wait?

**Comment by rvagg** [comment]
I think they did something about the semver warning in a recent 3.x didn't they? They should be pretty nice to use now.

@silverwind no point waiting, just keep in mind that we'll want to make room for another kind of download. It's unclear whether `nightly` should be off stable or master or both when we have this new channel. Currently nightlies are turned on for both, so you can get v5 and v6 from there.

**Comment by silverwind** [comment]
The version issue still persists with the npm provided in the package:

```
npm WARN engine node-zopfli@1.4.0: wanted: {"node":">=0.10"} (current: {"node":"6.0.0-nightly20160127fcae05e4b5","npm":"3.3.12"})
npm WARN engine handlebars@4.0.5: wanted: {"node":">=0.4.7"} (current: {"node":"6.0.0-nightly20160127fcae05e4b5","npm":"3.3.12"})
npm WARN engine postcss@5.0.14: wanted: {"node":">=0.12"} (current: {"node":"6.0.0-nightly20160127fcae05e4b5","npm":"3.3.12"})
```

**Comment by rvagg** [comment]
https://github.com/npm/npm/blob/master/CHANGELOG.md#better-node-pre-release-support

looks like it's in 3.6.0, which queued up in core waiting to be unblocked by some graceful-fs drama before it can land but will be in both v5 and v6 nightlies very soon

**Comment by silverwind** [comment]
Ah, nice to hear. I'd say we defer linking until then, so we don't have to deal with this headache.

**Comment by Knighton910** [comment]
:+1: @silverwind I'd be nice to have on the `downloads page`.

**Comment by stevemao** [comment]
Fixed by #559
